packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -2357,6 +2357,63 @@ Generated by [AVA](https://ava.li).
     ␊
     `
 
+## Prints outdated plugins installed in package.json
+
+> Snapshot 1
+
+    `␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build                                                 ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    > Version␊
+      @netlify/build 1.0.0␊
+    ␊
+    > Flags␊
+      debug: true␊
+      featureFlags: {}␊
+      repositoryRoot: /file/path␊
+      testOpts:␊
+        pluginsListUrl: /file/path␊
+        silentLingeringProcesses: true␊
+    ␊
+    > Current directory␊
+      /file/path␊
+    ␊
+    > Config file␊
+      /file/path␊
+    ␊
+    > Resolved config␊
+      plugins:␊
+        - inputs: {}␊
+          origin: config␊
+          package: netlify-plugin-contextual-env␊
+    ␊
+    > Context␊
+      production␊
+    ␊
+    > Available plugins␊
+    ␊
+    > Loading plugins␊
+       - netlify-plugin-contextual-env@1.0.0 from netlify.toml and package.json␊
+    ␊
+    > Outdated plugins␊
+       - netlify-plugin-contextual-env@1.0.0: latest version is 1.0.0␊
```

Comment:
Based on the fixture, shouldn't the ouput be: `netlify-plugin-contextual-env@0.2.0: latest version is 1.0.0` ?

Reply:
We normalize all versions in test snapshots to `1.0.0` to make test snapshots more stable.
In this particular case, this does make the test less useful (although checking that "Outdated plugins" is printing is useful).

Reply:
Ah cool, that explains it 😂 I was going "crazy" because the implementation looked correct, the fixture showed something but the snapshot showed other.

The test is definitely useful, and in reality it just means we can't validate the correct versions are mentioned, something I think we can all live with 😄 I'm wondering if it would be worth adding a comment for this? Or mentioning it in the test name? If you feel like it doesn't make sense happy to move on 👍

Reply:
Sounds good :+1: 
Done in 18432f6d61c0f520b5c094e4508f7c9fd788bf9a